content/2.developers/2.developer-tools/2.developer-cli/1.developer-cli.md

```diff
@@ -0,0 +1,78 @@
+---
+objectID: developers_developer-tools_developer-cli
+title: Developer CLI
+description: An overview of Archway Developer CLI commands, options and usage
+parentSection: Developers
+parentSectionPath: /developers
+---
+
+# Developer CLI
+
+The Developer CLI is the recommended tool to build, deploy, configure, instantiate, query, and interact with smart contracts. 
+
+There are currently two main releases:
+
+
+- [Archway CLI v2 (alpha)](/developers/developer-tools/developer-cli/developer-cli-v2)
+- [Archway CLI v1](/developers/developer-tools/developer-cli/developer-cli-v1)
+
+
+The CLI v2 introduces several improvements compared to v1:
+- Removal of **archwayd** as a dependency. This enables a simpler installation and flow, as well as enhancing speed and efficiency.
+- Commands have been re-organized and new sub-commands have been introduced.
+- Commands to query and withdraw rewards have been added, providing for a seamless experience for managing rewards.
+
+
+The <a href="https://www.npmjs.com/package/@archwayhq/cli/v/2.0.0-alpha.1" target="_blank" >Archway CLI v2</a> is currently in alpha testing. We recommend trying it out and submit any feedback in the <a href="https://github.com/archway-network/archway-cli/discussions/210" target="_blank" >Github Discussion</a>.
+
+## Dependencies
+
+### Nodejs / Npm
+
+Make sure to install node <a href="https://nodejs.org/en" target="_blank" >node </a> (v18.17.1 or above) and <a href="https://www.npmjs.com/" target="_blank" >npm</a>.
+
+
+### Docker
+
+Docker is needed as a dependency both in CLI v1 and v2, because of the <a href="https://github.com/CosmWasm/rust-optimizer" target="_blank" >rust optimizer</a>. This is because the optimizer needs to be configured in a way which can run in multiple platforms and generate a valid WASM binary that is supported by the chain is. Relying on Docker makes it way easier to control all platform and architecture dependencies required to produce a valid WASM file.
```

Comment:
```suggestion
Docker is needed as a dependency both in CLI v1 and v2, because of the <a href="https://github.com/CosmWasm/rust-optimizer" target="_blank" >rust optimizer</a>. This is because the optimizer needs to be configured in a way that can run on multiple platforms and generate a valid WASM binary that is supported by the chain. Relying on Docker makes it way easier to control all platform and architecture dependencies required to produce a valid WASM file.
```